src/main/java/fact/Utils.java

```diff
@@ -294,6 +295,41 @@ public static void mapContainsKeys(Data item, String... keys) {
 		}
 	}
 
+	/**
```

Comment:
Why reimplement this method?

Reply:
the orginal method needed String[], this allows List<String> it removes the need to convert a List into an array although I could change it into an collection instead of a list.

Reply:
changed it in the last to call the collection method if you want to use the String... keys method, so only one function is left